src/pages/DAO/index.tsx

```diff
@@ -166,55 +169,57 @@ const DAOPage: FunctionComponent = () => {
 						)}
 					</div>
 					<div className="dao__main">
-						<>
-							<HorizontalMenu
-								pages={menuEntries}
-								currentPage={page}
-								onChange={nextPage => {
-									push(`${pathname}?page=${nextPage}`)
+						<HorizontalMenu
+							pages={menuEntries}
+							currentPage={page}
+							onChange={nextPage => {
+								push(`${pathname}?page=${nextPage}`)
+							}}
+						/>
+						{isAdmin && page === "createNFT" && (
+							<CreateNFTForm
+								gnosisAddress={dao.gnosisAddress}
+								domains={user ? user.myDomains : []}
+								afterCreate={() => push(`${pathname}?page=collection`)}
```

Comment:
`afterCreate` is called right before setting the stage to "success", so it does the trick to not show success message and rather immediately redirect to collection